tests/datasets/test_sessions_processing.py

```diff
@@ -98,7 +102,7 @@ def query_runner(
                 ["started", ">", "2020-01-01 12:00:00"],
             ],
         },
-        "sessions_hourly_local",
+        materialized_storage.get_schema().get_table_name(),
```

Comment:
The read schema is already imported, why not simplify this to `read_schema.get_table_name()`

Reply:
Done.